### ISSUE TITLE
NPM module dependencies show security vulnerabilities

### ISSUE THREAD

**Issue by prantlf** [h2non/requireg#13]
[nsp](https://github.com/nodesecurity/nsp) maintained by [Node Security Platform](https://nodesecurity.io/) reports [Prototype Pollution](https://nodesecurity.io/advisories/612) in [deep-extend](https://npmjs.org/package/deep-extend):

```txt
> nsp check

(+) 1 vulnerability found
┌────────────┬────────────────────────────────────────────────────────────────────┐
│            │ Prototype Pollution                                                │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Name       │ deep-extend                                                        │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ CVSS       │ 2 (Low)                                                            │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Installed  │ 0.2.11                                                             │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Vulnerable │ <=0.5.0                                                            │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Patched    │ >=0.5.1                                                            │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Path       │ cs.core.ui@16.2.5 > requireg@0.1.7 > rc@1.0.3 > deep-extend@0.2.11 │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ More Info  │ https://nodesecurity.io/advisories/612                             │
└────────────┴────────────────────────────────────────────────────────────────────┘
```

**Comment by prantlf** [comment]
Thanks for merging my PR! Would you release a new version of your module, please? Thank you!

**Comment by h2non** [comment]
Yeah, will do tonight or tomorrow.

**Comment by h2non** [comment]
Sorry for the delay. It's now fixed in `v0.1.8`, also available on npm:
https://www.npmjs.com/package/requireg

**Comment by prantlf** [comment]
No problem. Thank you!
